packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.styles.tsx

```diff
@@ -0,0 +1,32 @@
+import { MediaMessageBorderColor, MessagePlacement } from './MediaMessage'
+
+export const getStyles = (
+  placement: MessagePlacement,
+  borderColor: MediaMessageBorderColor,
+  hasTime: boolean
+) => {
+  const { wrapperClasses, messageClasses } = getMessagePlacementClasses(placement, borderColor, hasTime);
+  const timeClasses = hasTime && 'mt-1 mb-2';
```

Comment:
Why do you need to give margin top to time and margin bottom to baseWrapper when hasTime? Could you just use a margin top on time whenever is time and avoid giving margin top to baseWrapper?

Reply:
done!